app/views/layouts/history.html.arb

```diff
@@ -0,0 +1,13 @@
+panel "History" do
+  table_for assigns[:versions] do
+    column ("Item") do |v| 
+        if v.item
+           link_to v.item.id, [:admin, v.item]
+        end
+    end
+    column ("Type") { |v| v.item_type.underscore.humanize }
+    column ("Modified at") { |v| v.created_at.to_s :long }
+    column ("Text") { |v| v.object.to_s }
```

Comment:
Interesting. it is a nice to have. just having the ability to go through history like this is already fun and useful.